packages/http-client-java/README.md

```diff
@@ -1,33 +1,58 @@
-# TypeSpec Java Emitter Library
+# @typespec/http-client-java
 
-This is a TypeSpec library that will emit a Java SDK from TypeSpec.
+TypeSpec library for emitting Java client from the TypeSpec REST protocol binding
 
-## Prerequisite
+## Install
+
+```bash
+npm install @typespec/http-client-java
+```
+
+## Usage
+
+### Prerequisite
 
 Install [Node.js](https://nodejs.org/) 20 or above. (Verify by running `node --version`)
 
 Install [Java](https://docs.microsoft.com/java/openjdk/download) 17 or above. (Verify by running `java --version`)
 
 Install [Maven](https://maven.apache.org/download.cgi). (Verify by running `mvn --version`)
 
-## Getting started
+## Usage
+
+1. Via the command line
+
+```bash
+tsp compile . --emit=@typespec/http-client-java
+```
+
+2. Via the config
+
+```yaml
+emit:
+  - "@typespec/http-client-java"
+```
 
-### Initialize TypeSpec Project
+The config can be extended with options as follows:
 
-Follow the [TypeSpec Getting Started](https://typespec.io/docs/) documentation to initialize your TypeSpec project.
+```yaml
+emit:
+  - "@typespec/http-client-java"
+options:
+  "@typespec/http-client-java":
+    option: value
+```
 
-Ensure that `npx tsp compile .` runs correctly.
+## Emitter options
 
-### Add Java Emitter http-client-java
+### `namespace`
```

Comment:
We should not document `namespace` for unbranded libraries. This option is only for Azure.

Reply:
OK. I will try get a PR to remove the `namespace` from EmitterOptions too.

Internally we can fallback to an automated resolving logic in code to replace reserved keyword `enum` to maybe `enumnamespace`